ui/main/src/app/business/templateGateway/cardTemplateGateway.ts

```
@@ -33,6 +34,7 @@ export class CardTemplateGateway {
     private static _functionToSendTemplateRenderingCompleteToTemplate: Function;
 
     private static _functionToGetUserResponseFromTemplate: Function;
+    private static _functionToEditCard: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)